static/src/javascripts/components/curl/curl-domReady.js

```diff
@@ -1,4 +1,4 @@
-(function(){/*
+window.require || (function(){/*
```

Comment:
An alternative might be to create a different `app.js` bundle just for interactive pages, which doesn't include `curl`. This is slightly less hacky, but maybe overkill?